test/functional/apps/management/_import_objects.ts

```diff
@@ -210,284 +208,5 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         expect(isSavedObjectImported).to.be(true);
       });
     });
-
-    describe('.json file', () => {
-      beforeEach(async function () {
-        await esArchiver.load('test/functional/fixtures/es_archiver/saved_objects_imports');
-        await kibanaServer.uiSettings.replace({});
-        await PageObjects.settings.navigateTo();
-        await PageObjects.settings.clickKibanaSavedObjects();
-      });
```

Comment:
You can remove all the `.json` export files that should no longer be used in `test/functional/apps/management/exports`

Reply:
Attempt 1: [`912c588` (#110738)](https://github.com/elastic/kibana/pull/110738/commits/912c588e9b2d181d0cebe280efbc11e056dcfca8)